src/org/parosproxy/paros/extension/history/ProxyListenerLog.java

```diff
@@ -144,6 +144,8 @@ private void createAndAddMessage(HttpMessage msg, int type) {
         extension.addHistory(historyRef);
 
         addToSiteMap(historyRef, msg);
+        
+        ProxyListenerLogEventPublisher.getPublisher().publishHrefAddedEvent(historyRef);

```

Comment:
This change requires a ZAP comment. I was wondering if we could use `HistoryReferenceEventPublisher` instead? Too generic maybe?

Reply:
I tried that, and it generated way too many events. This is more for someone trying to reproduce the History list, which is why I went for another publisher. But happy to reconsider...

Reply:
Right, fine as is :)